### DISCUSSION TITLE
Partial Coloring in q-expansion-item

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#16058] [CLI - PWA mode] [answered]
Dear Team , 
I would like to implement a partial background color as shown below in q-expansion-item. Please assist me in accomplishing this task.

In the design below, I have redacted certain industrial sensitive information in red to maintain confidentiality. Please disregard those sections.

![image](https://github.com/quasarframework/quasar/assets/8514576/4c8bb61f-92c4-4edd-8a55-bf4319fd390d)

**Comment by pdanpdan** [comment]
Make a codepen please with what you have tried

**Reply by jmriyaz84** [reply]
ok

**Reply by jmriyaz84** [reply]
@pdanpdan  , Please find codepen link below [https://codepen.io/jmriyaz84/pen/RwqxwYB?editors=1011](url)

Here I want only the background color for  '1 out 10' section should be in color remaining should be in white

![image](https://github.com/quasarframework/quasar/assets/8514576/fcd835f3-c237-4808-8f51-d84533c98118)

Like below design
![image](https://github.com/quasarframework/quasar/assets/8514576/05ea3f12-1fee-4ec1-9593-13b7e09d0245)

**Comment by pdanpdan** [comment, marked as answer]
Something like this https://codepen.io/pdanpdan/pen/LYXeEPd?editors=1101?

**Reply by jmriyaz84** [reply]
Great ,  working well , Thanks a lot for quick support .
